package.json

```diff
@@ -83,161 +74,57 @@
-    "postcss-loader": "3.0.0",
     "raf": "3.4.0",
     "resolve": "1.8.1",
-    "sass-loader": "7.1.0",
-    "source-map-loader": "0.2.4",
     "storybook-readme": "4.0.2",
-    "stylelint": "9.6.0",
-    "stylelint-config-recommended-scss": "3.2.0",
-    "stylelint-scss": "3.3.2",
-    "time-fix-plugin": "2.0.5",
-    "ts-jest": "23.10.4",
-    "ts-loader": "5.2.2",
     "ts-node": "7.0.1",
-    "tsconfig-paths-webpack-plugin": "3.2.0",
-    "tslint": "5.11.0",
-    "tslint-config-prettier": "1.15.0",
-    "tslint-eslint-rules": "5.4.0",
-    "tslint-react": "3.6.0",
-    "typescript": "3.1.1",
-    "uglifyjs-webpack-plugin": "2.0.1",
-    "url-loader": "1.1.2",
-    "webpack": "4.23.0",
-    "webpack-config-utils": "2.3.1",
-    "webpack-dev-middleware": "3.4.0",
-    "webpack-dev-server": "3.1.10",
-    "webpack-flush-chunks": "2.0.3",
-    "webpack-hot-middleware": "2.24.3",
-    "webpack-hot-server-middleware": "0.5.0",
-    "webpack-log": "2.0.0",
-    "webpack-manifest-plugin": "2.0.4",
-    "webpack-merge": "4.1.4",
-    "webpack-node-externals": "1.7.2"
+    "webpack": "4.25.1",
+    "webpack-flush-chunks": "2.0.3"
   },
   "devEngines": {
     "node": "8.x || 9.x || 10.x"
   },
   "scripts": {
-    "build": "run-p build-app build-story",
-    "build-app": "ts-node --files lib/build.ts",
-    "build-story": "build-storybook -c .storybook -o build/public/docs",
-    "clean": "rm -rf ./build",
+    "build": "run-p build:app build:story",
+    "build:app": "pixeloven-tasks build",
+    "build:story": "build-storybook -c .storybook -o build/public/docs",
     "deploy": "node build/server.js",
-    "lint": "run-p -l lint-app lint-scss",
-    "lint-fix": "run-p -l lint-app-fix lint-scss-fix",
-    "lint-app": "tslint --project .",
-    "lint-app-fix": "tslint --fix --project .",
-    "lint-scss": "stylelint src/**/*.scss --syntax scss",
-    "lint-scss-fix": "stylelint src/**/*.scss --syntax scss --fix",
+    "lint": "pixeloven-cli lint:ts && pixeloven-cli lint:scss",
+    "lint:ts": "pixeloven-cli lint:ts",
+    "lint:scss": "pixeloven-cli lint:scss",
+    "prebuild": "rm -rf ./build",
     "precommit": "lint-staged",
-    "start": "run-p start-ts start-story",
-    "start-ts": "ts-node --files lib/start.ts",
-    "start-story": "start-storybook --quiet -s ./public -p 9001 -c .storybook",
-    "test": "jest --watch --env=jsdom",
-    "test-coverage": "jest --coverage --env=jsdom"
+    "pretest": "pixeloven-cli test:clean",
+    "pretty": "pixeloven-cli pretty",
+    "pretty:ts": "pixeloven-cli pretty:ts",
+    "pretty:scss": "pixeloven-cli pretty:scss",
+    "start": "run-p start:app start:story",
+    "start:app": "pixeloven-tasks serve",
+    "start:story": "start-storybook --quiet -s ./public -p 9001 -c .storybook",
+    "test": "pixeloven-cli test",
+    "test:watch": "pixeloven-cli test:watch"
   },
   "lint-staged": {
-    "lib/**/*.{ts,tsx}": [
-      "yarn lint-app-fix",
+    "src/**/*.{scss,ts,tsx}": [
```

Comment:
Why do we need lint these files as a group here and split out by file type below?

Reply:
I had them all together but then realized that husky pipes the file names to the script and so by splitting them we can better optimize the jobs.

Reply:
Do we need them grouped at all then or can this first item be removed?